server/neptune/lyft/activities/github.go

```diff
@@ -55,6 +57,18 @@ func (a *Github) GithubListPRs(ctx context.Context, request ListPRsRequest) (Lis
 	}, nil
 }
 
+func IsPRAutomated(pr *github.PullRequest) bool {
+	if pr.Labels == nil {
```

Comment:
https://pkg.go.dev/github.com/google/go-github/v45/github#PullRequest
```
Labels              []*[Label](https://pkg.go.dev/github.com/google/go-github/v45/github#Label)              `json:"labels,omitempty"`
```

it's better to check zero length for a slice instead of nil, because there can be empty allocated slice.